templates/member/member_detail.html

```diff
@@ -185,6 +185,12 @@
                     </div>
                 </div>
                 <div class="card-body">
+                    <div class="row">
+                        <div class="d-none d-sm-inline col-sm-3">{% trans "Show gig time on schedule" %}:</div>
+                        <div class="d-none d-sm-inline col-sm-9">{% include "base/tf.html" with value=member.preferences.agenda_show_time %}</div>
+                        <div class="col-12 d-sm-none">{% trans "Show gig time on schedule" %}: {% include "base/tf.html" with value=member.preferences.agenda_show_time %}</div>
+                    </div>
+                    <br>
```

Comment:
Interesting that it seems like the code was there, but the preference was not exposed. 

Do you know the use case for *not* showing gig times? Would it make more sense to just always show them?